examples/cookies/client.go

```diff
@@ -0,0 +1,65 @@
+// +build ignore
+
+package main
+
+import (
+	"fmt"
+
+	"github.com/erikdubbelboer/fasthttp"
+	"github.com/themester/fcookiejar"
+)
+
+func main() {
+	req, res := fasthttp.AcquireRequest(), fasthttp.AcquireResponse()
+	req.SetRequestURI("http://localhost:8080")
+
+	err := fasthttp.Do(req, res)
+	if err != nil {
+		panic(err)
+	}
+
+	// We can achieve cookies by different ways.
+	// 1. Using 'key' parameter. Fasthttp will search using key in cookie field.
+	cookie := fasthttp.AcquireCookie()
+	// do not forget to release
+	defer fasthttp.ReleaseCookie(cookie)
+	cookie.SetKey("key")
+
+	res.Header.Cookie(cookie)
+	fmt.Printf("You cookie is '%s' found using '%s' key\n", cookie.Cookie(), cookie.Key())
```

Comment:
Your cookie is '%s' found using '%s' as key

Reply:
arggghhh... sorry for my typos